### PR TITLE
Add EKS-D dev build 1.25 support

### PR DESCRIPTION
**Pull request by taneyland**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/kubernetes-sigs/kind/buildspecs/batch-build.yml [hunk 1]
```diff
@@ -13,7 +13,6 @@
 # limitations under the License.
 
 # This file was generated by running `make generate`. DO NOT EDIT
-
```
Comment (mitalipaygude) [inline projects/kubernetes-sigs/kind/buildspecs/batch-build.yml:16]: unintentional change?

File: projects/aws/bottlerocket-bootstrap/buildspecs/batch-build.yml [hunk 2]
```diff
@@ -13,7 +13,6 @@
 # limitations under the License.
 
 # This file was generated by running `make generate`. DO NOT EDIT
-
```
Comment (mitalipaygude) [inline projects/aws/bottlerocket-bootstrap/buildspecs/batch-build.yml:16]: unintentional?

File: projects/kubernetes/cloud-provider-vsphere/buildspecs/batch-build.yml [hunk 3]
```diff
@@ -13,7 +13,6 @@
 # limitations under the License.
 
 # This file was generated by running `make generate`. DO NOT EDIT
-
```
Comment (mitalipaygude) [inline projects/kubernetes/cloud-provider-vsphere/buildspecs/batch-build.yml:16]: same here?

File: projects/aws/bottlerocket-bootstrap/buildspecs/batch-build.yml (hunk 2 above)
Comment (taneyland) [reply projects/aws/bottlerocket-bootstrap/buildspecs/batch-build.yml:16]: no that was added when i ran `make generate`

**Review by TerryHowe — APPROVED** [review]
/lgtm

**Pull request merged**